### ISSUE TITLE
Resource timeline view grid doesn't line up on wide screens in Chrome

### ISSUE THREAD

**Issue by mchapman** [fullcalendar/fullcalendar-scheduler#505]
Please see https://codepen.io/anon/pen/pYWYBg (probably only visible on a 4k screen maximised window)

Towards the end of the day you get problems like this:
![Screenshot from 2019-03-12 11-31-57](https://user-images.githubusercontent.com/373462/54196909-7b9cf300-44ba-11e9-995f-de3cb560e7b9.png)

**Comment by iiibbbmmm** [comment]
Confirmed on dual monitors, with 3840px width.

**Comment by acerix** [comment]
Thanks for the report.  I can see the issue in Chrome on a smaller screen by zooming out to 25%, but don't see the issue in Firefox.  Are you also using Chrome?

**Comment by mchapman** [comment]
I am, yes. Afraid I didn't test it anywhere else

**Comment by zachjharris** [comment]
Setting the `slotWidth` to any string seems to fix the issue. Looking at [this](https://github.com/fullcalendar/fullcalendar-scheduler/blob/be78dc5dabaa3851049a2bef2e83a09965e2b678/src/timeline/TimeAxis.ts#L118) now to see if I can figure out the issue.

**Comment by dokithonon** [comment]
I had a similar problem due to a custom css rule that I have added

.fc-head {
  font-size: 10px;
}

Removing this has fixed the issue.

![image](https://user-images.githubusercontent.com/1856638/55405689-eb9f1600-555a-11e9-8a64-2cff47d1cf3f.png)

**Comment by arshaw** [comment]
i can also repro this if i zoom out a lot on Chrome

**Comment by nsorin** [comment]
I noticed something which seems to cause this issue:
The width of the header cells seems to be regulated by a `<colgroup>` element which contains one `<col>` element for each slot.
In v3, with standard settings (one slot = 30 minutes, 24 hours displayed, so 48 slots in total),
there is one `<col>` element with a set `width` for each slot, except the last one which has no width:

``` html
<colgroup>
    <col style="width: 35px;"> <!-- 47 times -->
    <col>
</colgroup>
```

In v4, however, there are not enough `<col>` elements (only 23 instead of 47). Adding the col elements manually seems to solve the issue. So I guess there could be a problem in the HTML generation.

**Comment by arshaw** [comment]
fixed in v4.1.0

@nsorin your diagnosis was exactly correct!
